.github/sync-packages.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+if ('cli' !== PHP_SAPI) {
+    echo "This script can only be run from the command line.\n";
+    exit(1);
```

Comment:
script should be removed I guess?

Reply:
It should be run on newest branches for new components. And can be used to update the files quickly.